eth/api.go

```diff
@@ -607,3 +607,13 @@ func (api *PrivateDebugAPI) GetAccessibleState(from, to rpc.BlockNumber) (uint64
 	}
 	return 0, fmt.Errorf("No state found")
 }
+
+type gcPolicy struct {
+	Archive        bool
```

Comment:
It's nice to surface this to the user, so he/she can set `{ "archive"=true} `, but not so nice to just ignore archive and only care about FlushFrequency it later on :trollface: